.github/workflows/golangci-lint.yml

```diff
@@ -18,5 +18,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.57.2
-          skip-pkg-cache: true
+          version: v1.61.0
```

Comment:
You can just use `v1.61` here so we don't need to worry about the patch versions in the GH workflows.